provider/kubernetes/annotations.go

```diff
@@ -9,6 +9,10 @@ const (
 	annotationKubernetesAuthRealm                 = "ingress.kubernetes.io/auth-realm"
 	annotationKubernetesAuthType                  = "ingress.kubernetes.io/auth-type"
 	annotationKubernetesAuthSecret                = "ingress.kubernetes.io/auth-secret"
+	annotationKubernetesForwardAuthURL            = "ingress.kubernetes.io/auth-forward-url"
```

Comment:
In keeping in line with other ingress controllers, perhaps `auth-url` should be used instead: (https://github.com/kubernetes/ingress-nginx/blob/master/docs/user-guide/nginx-configuration/annotations.md)